### PR TITLE
Fixed conversion warning with level in compress_bound unit tests.

### PR DESCRIPTION
**Pull request by nmoinvaz**
```
  test_compress_bound.cc(43,1): warning C4267: 'argument': conversion from 'size_t' to 'int32_t', possible loss of data
```

### CODE REVIEW

**Review by Dead2 — APPROVED** [review]
LGTM

File: test/test_compress_bound.cc [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 class compress_bound_variant : public testing::TestWithParam<z_size_t> {
```
Comment (mtl1979) [inline test/test_compress_bound.cc]: Shouldn't this use `int32_t`too?

Comment (nmoinvaz) [reply test/test_compress_bound.cc]: Yes you are right.

Comment (nmoinvaz) [reply test/test_compress_bound.cc]: Fixed.

**Pull request merged**